_includes/javadocs.html

```diff
@@ -3,6 +3,7 @@
   <div class="container">
     <h4>API Snapshots:
       <a href="/api/java/snapshot/apidocs/index.html">Java Core</a>,
+      <a href="https://apache.github.io/datasketches-cpp/master/">C++ Core</a>,
```

Comment:
We should change this to the latest stable release once we have a release with proper docs.